### PR TITLE
Remove traits from template.json where unnecessary

### PR DESCRIPTION
**Pull request by stwlam**

### CODE REVIEW

File: static/template.json [hunk 1]
```diff
@@ -481,19 +481,30 @@
...
                 "schema": {
                     "version": null,
                     "lastMigration": null
                 }
             },
+            "traits": {
+                "traits": {
+                    "value": []
+                }
+            },
```
Comment (TikaelSol) [inline static/template.json:495]: traits.traits?

File: static/template.json [hunk 2]
```diff
@@ -481,19 +481,30 @@
...
+            "rarity": {
+                "traits": {
+                    "rarity": "common"
+                }
+            },
```
Comment (TikaelSol) [inline static/template.json:500]: rarity.traits.rarity?

File: static/template.json [hunk 3]
```diff
@@ -481,19 +481,30 @@
...
             "physical": {
+                "level": {
+                    "value": 0
+                },
                 "traits": {
+                    "value": [],
+                    "rarity": "common",
                     "otherTags": []
                 },
```
Comment (TikaelSol) [inline static/template.json:509]: physical.level? physical.traits.rarity?

File: static/template.json (hunk 1 above)
Comment (stwlam) [reply static/template.json:495]: These are template fields

**Pull request merged**